### PR TITLE
[ARM CPU] Fix AVGPool tests

### PR DESCRIPTION
**Pull request by allnes**
The combination of parameters: NCHW + CEIL gives an accuracy problem in AvgPool. One workaround is to disable the ACL executor for these parameters. Then OneDNN will run this case in ACL backend as reorder -> NHWC -> reorder and combination NHWC + CEIL without accuracy drop.

### CODE REVIEW

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp [hunk 1]
```diff
@@ -42,8 +42,9 @@ bool AclPoolingExecutor::isSupported(const TensorInfo& srcTensorInfo,
         DEBUG_LOG("Unknown pooling algorithm: ", static_cast<int>(poolingAttrs.algorithm));
         return false;
     }
-    DimensionRoundingType round = (poolingAttrs.rounding == op::RoundingType::CEIL) ?
-                                   DimensionRoundingType::CEIL : DimensionRoundingType::FLOOR;
+
+    if (dataLayout == arm_compute::DataLayout::NCHW && poolingAttrs.rounding == op::RoundingType::CEIL) return false;
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp:46]: how does CEIL and NHWC work?

Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp:46]: Please add a comment with clarification

Comment (allnes) [reply src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp:49]: @EgorDuplensky done

**Review by EgorDuplensky — APPROVED** [review]
Please add the same explanation you added as a comment to the code, into PR message

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp (hunk 1 above)
Comment (allnes) [reply src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp:49]: @ilya-lavrenov I corrected rounding, for other cases.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**